File: Resueltos_por_competidor/Santiago_Ortiz/Translation.java
import java.util.Scanner;

/*
 * Autor: Darkovski7   
 * Problema: Translation
 * Juez Online: Codeforces
 * Veredicto: Accepted 
 * Url: https://codeforces.com/problemset/problem/41/A
 */

public class Translation {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		String s = sc.next();
		String s2 = sc.next();
		
		String inverso = new StringBuilder(s).reverse().toString();
		
		if (inverso.equals(s2)) {
			System.out.println("YES");
		} else {
			System.out.println("NO");
		}
	}

}

File: Resueltos_por_competidor/Carlos_Cobaleda/Translation.cpp
/*   
Autor: Carlos Eduardo Cobaleda Moreno
Problema: Translation
Juez online: Codeforces
Veredicto: Accepted
URL: https://codeforces.com/contest/41/problem/A
*/

#include <bits/stdc++.h>
using namespace std;
 
int main() {
	string s,t; cin >> s >> t;
	reverse(s.begin(), s.end());
	if(s == t) {
        cout << "YES" << endl;
	} else {
	cout << "NO" << endl;
    }
	return 0;
}
